packages/ckeditor5-image/src/image/utils.js

```diff
@@ -112,6 +112,30 @@ export function isImage( modelElement ) {
 	return isInlineImage( modelElement ) || isBlockImage( modelElement );
 }
 
+/**
+ * Checks if the provided view element represents an inline image.
+ *
+ * Also, see {@link module:image/image/utils~isImageWidget}.
+ *
+ * @param {module:engine/view/element~Element} element
+ * @returns {Boolean}
+ */
+export function isInlineViewImage( element ) {
+	return !!element && element.is( 'element', 'img' );
+}
```

Comment:
> This util is very generic so I can image cases when someone would use it on the `img` element inside a `figure` and it would give an invalid result. Maybe we should check if this element's parent is not a figure?

TBH I don't see anything wrong with using it for the inner structure of a block widget; it already happens in this PR in `getViewImageFromWidget()`.

Reply:
:+1: